### DISCUSSION TITLE
How to open MP3 or MP4 File in new browser window ?

### DISCUSSION THREAD

**Discussion by Miami15** [DirectoryLister/DirectoryLister#767] [Help & Support] [answered]
Hi Chris, Hi All !
What do I have to write in which file if I want to open a .mp3 ending file (or similar .mp4 also)  in a new browser window/tap ?
(otherwise people will close the listening and the whole directory is also gone...)

Thank you in advance for your recent help !
Sincerely
Franz

**Comment by PHLAK** [comment, marked as answer]
This is not possible via configuration. You would need to modify the source directly (and compile it manually) to add `target="_blank"` to the anchor tags.
